applications/ParticleMechanicsApplication/custom_strategies/schemes/mpm_residual_based_bossak_scheme.hpp

```diff
@@ -225,12 +225,12 @@ class MPMResidualBasedBossakScheme
             if (!(i->pGetDof(DISPLACEMENT_X)->IsFixed()))
                 r_current_displacement[0] = 0.0;
             else
-                r_current_displacement[0]  = r_previous_displacement[0];
+                r_current_displacement[0]  = 0.0;
 
             if (!(i->pGetDof(DISPLACEMENT_Y)->IsFixed()))
                 r_current_displacement[1] = 0.0;
             else
-                r_current_displacement[1]  = r_previous_displacement[1];
+                r_current_displacement[1]  = 0.0;
```

Comment:
Are you sure abt this one?
Are the tests running?

Reply:
I need to reset the current displacement for my case especially if the dofs are fixed. Perhaps @iiaconeta or @bodhinandach know why there is this distinction and whether we still need it. 
All tests are running.

Reply:
@VeronikaSinger Can you try to test where you apply prescribed displacement to the nodes for dynamic cases?

Reply:
have you got an example where prescribed displacement is used?

Reply:
I resolved this problem